### ISSUE TITLE
 "Discover Mode" - Display Random Videos on Initial Page Load

### ISSUE THREAD

**Issue by GoofyExploit** [Open-Source-Chandigarh/MeTube#5]
When users first visit the MeTube, they are presented with a blank page. To enhance the user experience and encourage exploration, we propose implementing a "Discover Mode" feature. This feature will display a selection of random YouTube videos on the initial page load, piquing the user's interest and suggesting content they might enjoy.

Goals:

   1. Increase user engagement and exploration of the MeTube.
   2. Provide a more dynamic and interactive initial page load experience.
   3. Encourage users to discover new content and explore different topics.

Let me know if i can proceed with this idea and start contributing.

**Comment by kunalbhatia2601** [comment]
That's a Great Ideaaa, Go with it!!!

**Comment by MITARPALGROVER** [comment]
@kunalbhatia2601 @GoofyExploit  but the main purpose of this metube was to make a distraction free interface 
it was made like to just search what you need coz like in youtube when we enter so much of videos displayed on the loading page and sometimes instead of searching what we want to search, we get distracted towards those videos which are displayed on the screen

**Comment by GoofyExploit** [comment]
> @kunalbhatia2601 @GoofyExploit but the main purpose of this metube was to make a distraction free interface it was made like to just search what you need coz like in youtube when we enter so much of videos displayed on the loading page and sometimes instead of searching what we want to search, we get distracted towards those videos which are displayed on the screen

yeah, but if anyone is just here to spend some time then it is great, otherwise i can also add a new feature that lets us enable and disable this feature

**Comment by MITARPALGROVER** [comment]
ok that's really a good idea to add a new feature

**Comment by perkyPearl** [comment]
It would be better to have a toggle where user can choose to see the random Video Suggestion on the Homepage or not. Or one more thing, And if it time based. It would be Great.

**Comment by MITARPALGROVER** [comment]
@perkyPearl  yeah nice idea
